.evergreen/config_generator/components/sasl/nossl.py

```diff
@@ -22,8 +22,8 @@
 ]
 
 TEST_MATRIX = [
-    ('ubuntu1604', 'gcc', None, 'off', ['noauth'], ['server', 'replica', 'sharded'], ['3.6',                                     ]),
-    ('ubuntu1804', 'gcc', None, 'off', ['noauth'], ['server', 'replica', 'sharded'], [       '4.0', '4.2', '4.4', '5.0', 'latest']),
+    ('ubuntu1604', 'gcc', None, 'off', ['noauth'], ['server', 'replica', 'sharded'], ['3.6',                                            ]),
```

Comment:
```suggestion
    ('ubuntu1604', 'gcc', None, 'off', ['noauth'], ['server', 'replica', 'sharded'], ['3.6',                                                   ]),
```
Whitespace alignment.